lib/linguist/languages.yml

```diff
@@ -625,6 +625,7 @@ CartoCSS:
   language_id: 53
 Ceylon:
   type: programming
+  color: "#563a08"
```

Comment:
Try `#dfa535` instead. Much closer to the colour of Ceylon's logo than `#563a08`.